engine/gamesys/src/gamesys/resources/res_sound_data.cpp

```diff
@@ -124,15 +125,20 @@ namespace dmGameSystem
 
     static void AddChunk(HResourceChunkCache cache, dmhash_t path_hash, uint8_t* data, uint32_t size, uint32_t offset)
     {
+        int flags = offset == 0 ? RESOURCE_CHUNK_CACHE_NO_EVICT : RESOURCE_CHUNK_CACHE_DEFAULT;
```

Comment:
In our current case, the "offset == 0" means the initial chunk of a file (the part that contains the header of a file)

Reply:
Maybe this could be an actual comment in the code? It makse sense, but could be easily missed why we would need the first chunk 🤔